external/loaders/loaders/batches/save.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+from loaders._config import BatchesLoader
+import yaml
+import os.path
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        description=(
+            "Save a BatchesLoader configuration locally "
+            "as a directory of netCDF files."
+        )
+    )
+    parser.add_argument(
+        "data_config",
+        type=str,
+        help="path of loaders.BatchesLoader training data yaml file",
+    )
+    parser.add_argument(
+        "output_path",
+        type=str,
+        help="local directory to save data as numbered netCDF files",
+    )
+    return parser
+
+
+def main(data_config: str, output_path: str):
+    with open(data_config, "r") as f:
+        config = yaml.safe_load(f)
+    loader = BatchesLoader.from_dict(config)
+    batches = loader.load_batches()
+    for i, batch in enumerate(batches):
+        out_filename = os.path.join(output_path, f"{i:05}.nc")
```

Comment:
Nit: I assume some configs can be pretty slow - add some output to give the user a sense of how long this is going to take?

Reply:
Ah great idea, I like a printout of the batch number in this case. Will add.